interface/super/edit_list.php

```diff
@@ -43,6 +43,11 @@
     die(xlt('Not authorized'));
 }
 
+//Limit variables for filter
+$list_from = ( isset($_GET["list_from"]) ? intval($_GET["list_from"]) : (isset($_POST["list_from"]) ? intval($_POST["list_from"]) : 0) );
+$list_to   = ( isset($_GET["list_to"]) ? intval($_GET["list_to"]) : (isset($_POST["list_to"]) ? intval($_POST["list_to"]) : 10) );
```

Comment:
Couldn't this be simplified if use $_REQUEST ?

Reply:
You are right.